optuna/samplers/_cmaes.py

```diff
@@ -142,6 +142,10 @@ def objective(trial):
                 to set this flag :obj:`True` when the :class:`~optuna.pruners.HyperbandPruner` is
                 used. Please see `the benchmark result
                 <https://github.com/optuna/optuna/pull/1229>`_ for the details.
+
+    Raises:
+        ValueError:
+            If ``restart_strategy`` is not 'ipop' or :obj: `None`.
```

Comment:
A nit, but could you remove a white space between `:obj:` and `None`?
```suggestion
            If ``restart_strategy`` is not 'ipop' or :obj:`None`.
```

![image](https://user-images.githubusercontent.com/3255979/97542089-83036e00-1a09-11eb-9647-1ebe6fe40a4f.png)

Reply:
oh, sorry, thank you!

Reply:
Sorry should've noticed as well. Thanks @toshihikoyanase and @yuk1ty .

Reply:
No problem at all. We employ two-approval rule to help each other.